### ISSUE TITLE
Publish to Maven central

### ISSUE THREAD

**Issue by whiskeysierra** [ClickerMonkey/TrieHard#5]
Have you considered publishing this library to maven central, to make it more accessible?

**Comment by ClickerMonkey** [comment]
Yes, but after wasting too many hours trying to do so I gave up. I'm out of
the Java world and am unfamiliar with Maven.
